plugins/aws-s3-storage/src/deleteKeyPrefix.ts

```diff
@@ -11,11 +11,12 @@ export function deleteKeyPrefix(s3: S3, options: DeleteKeyPrefixOptions, callbac
     if (err) {
       callback(convertS3Error(err));
     } else if (data.KeyCount) {
+      const objectsToDelete: S3.ObjectIdentifierList =
+        data.Contents !== undefined ? data.Contents.map(s3Object => ({ Key: s3Object.Key as S3.ObjectKey })) : [];
```

Comment:
I think the `undefined` checking could be removed (only `data.Contents ?` should work).

Reply:
Changed it. It's less explicit but works of course. If this is the way to go and alignes with your coding conventions and style, that's totally legit.